### PR TITLE
docs - remove deprecated-in-v5 config props and vars

### PR DESCRIPTION
**Pull request by lisakowen**
doc updates for the deprecated-in-v5, removed-in-v6 features/properties:
- remove some notes about the properties
- removed old hadoop profile names
- call out with step in 5->6 upgrade guide

related: update the default value description of pxf.service.user.impersonation

doc review site link for mods upgrade - scroll down to step 7:http://docs-lisa-pxf6-review.cfapps.io/pxf/6-0/using/upgrade_5_to_6.html#pxfup

### CODE REVIEW

**Review by ashuka24 — APPROVED** [review]

**Review by frankgh — APPROVED** [review]

File: docs/content/access_hdfs.html.md.erb [hunk 1]
```diff
@@ -103,21 +103,21 @@ The PXF Hadoop connectors provide built-in profiles to support the following dat
...
+| HDFS | delimited [text with quoted linefeeds](hdfs_text.html#profile_textmulti) |
+| HDFS | [Avro](hdfs_avro.html) | hdfs:avro |
+| HDFS | [JSON](hdfs_json.html) | hdfs:json |
+| HDFS | [Parquet](hdfs_parquet.html) | hdfs:parquet |
+| HDFS | AvroSequenceFile | hdfs:AvroSequenceFile |
+| HDFS | [SequenceFile](hdfs_seqfile.html) | hdfs:SequenceFile |
+| [Hive](hive_pxf.html) | stored as TextFile | Hive, [HiveText](hive_pxf.html#hive_text) |
+| [Hive](hive_pxf.html) | stored as SequenceFile | Hive |
+| [Hive](hive_pxf.html) | stored as RCFile | Hive, [HiveRC](hive_pxf.html#hive_hiverc) |
+| [Hive](hive_pxf.html) | stored as ORC | Hive, [HiveORC](hive_pxf.html#hive_orc), HiveVectorizedORC |
```
Comment (denalex) [inline docs/content/access_hdfs.html.md.erb:118]: product question: should we rename these as `hive`, `hive:orc`, `hive:rc` with vectorized being a user option ? @frankgh ?

**Review by denalex — APPROVED** [review]

File: docs/content/access_hdfs.html.md.erb (hunk 1 above)
Comment (frankgh) [reply docs/content/access_hdfs.html.md.erb:118]: that makes sense, we wanted to do that at some point, so it makes sense to do it now

File: docs/content/upgrade_5_to_6.html.md.erb [hunk 2]
```diff
@@ -106,7 +106,16 @@ After you install the new version of PXF, perform the following procedure:
 
 1. **If you are upgrading to PXF version 6.0**:
 
-    1. step 1
+    1. Ensure that you no longer reference previously deprecated features that are removed in PXF 6.0:
```
Comment (dyozie) [inline docs/content/upgrade_5_to_6.html.md.erb:109]: are -> were

?  Maybe, to work for later releases.

File: docs/content/upgrade_5_to_6.html.md.erb [hunk 3]
```diff
@@ -106,7 +106,16 @@ After you install the new version of PXF, perform the following procedure:
...
+
+        | Deprecated Feature | Use Instead |
+        | -------------------|-------------|
+        | Hadoop profile names | hdfs:\<profile> as noted [here](access_hdfs.html#hadoop_connectors) |
+        | `jdbc.user.impersonation` property | `pxf.service.user.impersonation` property in the [jdbc&#8209;site.xml](jdbc_cfg.html#jdbcimpers) server configuration file |
+        | PXF_KEYTAB configuration property | `pxf.service.kerberos.keytab` property in the [pxf&#8209;site.xml](cfg_server.html#pxf-site) server configuration file |
```
Comment (dyozie) [inline docs/content/upgrade_5_to_6.html.md.erb:115]: Not sure why the all-caps property names aren't in `code` format here.  But if that's the convention used throughout it's fine.

**Review by dyozie — APPROVED** [review]

**Pull request merged**